bakerydemo/templates/blog/blog_index_page.html

```diff
@@ -2,48 +2,34 @@
 {% load wagtailcore_tags navigation_tags wagtailimages_tags %}
 
 {% block content %}
-    {% include "base/include/header-index.html" %}
+    {% if not tag %}
+        {% include "base/include/header-index.html" %}
+    {% endif %}
 
     <div class="container">
         {% if tag %}
             <div class="row">
                 <div class="col-md-12">
-                    <p>Viewing all blog posts by <span class="outline">{{ tag }}</span></p>
+                    <h1 class="index-header__title">Blog</h1>
+                </div>
+                <div class="col-md-12">
+                    <p class="index-header__introduction">Viewing all blog posts sorted by the tag <span class="blog-tags__tag">{{ tag }}</span>.</p>
                 </div>
             </div>
         {% endif %}
 
         {% if page.get_child_tags %}
-            <ul class="blog-tags tags list-inline">
+            <ul class="blog-tags">
                 {% for tag in page.get_child_tags %}
-                    <li><a href="{{ tag.url }}">{{ tag }}</a></li>
+                    <li><a class="blog-tags__pill" href="{{ tag.url }}">{{ tag }}</a></li>
```

Comment:
suggestion: add an `aria-label` to this to give the user more context about what's about to happen - at the moment screenreader announces "link, fermentation"

Reply:
Good idea, thank you